### ISSUE TITLE
Accessibility:List: Extra information “loading” is conveyed, when ‘Show More’ button is accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3483]
Description: Extra information “loading” is conveyed, when ‘Show More’ button is accessed. 

Expected: Extra information should not be conveyed. Only relevant information should be read out when each & every UI is accessed

screenshot:

![image](https://user-images.githubusercontent.com/32538291/94648783-def6ac00-0310-11eb-9b5e-6cb06d67412d.png)
